src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -994,4 +285,6 @@ def connect_to_fluent(
         ip, port, password = _get_server_info(server_info_file_name, ip, port, password)
         watchdog.launch(os.getpid(), port, password, ip)
 
-    return new_session(fluent_connection=fluent_connection)
+    return new_session(
```

Comment:
While testing `connect_to_fluent` found that we are getting following error.

![image](https://github.com/ansys/pyfluent/assets/106588300/4a765cb9-49ab-4441-991a-702f52b3974d)

We have resolved this now by passing `remote_file_handler` here.